ros/api/v1/hosts.py

```diff
@@ -182,6 +201,14 @@ def build_system_filters():
             filters.append(System.state.in_(modified_states))
         else:
             filters.append(System.state.in_(SYSTEM_STATES_EXCEPT_EMPTY))
+        if operating_systems := request.args.getlist('os'):
+            modified_operating_systems = []
+            for os in operating_systems:
+                os = os.upper()
+                if os not in OS_VERSIONS.keys():
```

Comment:
Here, we're getting `os` as `RHEL X.Y` and `os_versions` keys as 'X.Y'. So, this condition is always true. 

![Screenshot from 2022-04-07 13-32-33](https://user-images.githubusercontent.com/61837065/162150625-21f11449-4a5f-45ab-beef-5b423cedbc21.png)

Reply:
The motivation here is to pass `/systems?os=7.0`

Reply:
But on openapi it's `RHEL 7.0`

Reply:
Updating it in openapi spec here: https://github.com/RedHatInsights/ros-backend/pull/205/files#diff-80c724753dbc2f811ac54a576c78bb6d32e37e6e76fd8e138252a3a151051992R641-R656

Reply:
can you retest now?

Reply:
But getting error for every value
![Screenshot from 2022-04-07 13-32-33](https://user-images.githubusercontent.com/61837065/162156310-c1bb2dbd-6eb4-45ce-ad60-62f16e686328.png)